backend/app/services/chat/trigger/core.py

```
@@ -505,3 +553,665 @@ async def _stream_chat_response(
         span_manager.exit_span()
 
         db.close()
+
+
+async def _stream_with_http_adapter(
+    stream_data: StreamTaskData,
+    message: str,
+    model_config: dict,
+    system_prompt: str,
+    ws_config: Any,
+    extra_tools: list,
```

Ruff (0.14.10)

Unused function argument: `extra_tools`

(ARG001)